src/asciidoc/web-mvc.adoc

```diff
@@ -3822,8 +3822,8 @@ response accordingly. By default the `DispatcherServlet` registers the
 When the status of the response is set to an error status code and the body of the
 response is empty, Servlet containers commonly render an HTML formatted error page. To
 customize the default error page of the container, you can declare an `<error-page>`
-element in `web.xml`. Up until Servlet 3, that element had to be mapped to a specific
-status code or exception type. Starting with Servlet 3 an error page does not need to be
+element in `web.xml`. Up until Servlet 3.0, that element had to be mapped to a specific
+status code or exception type. In a Servlet 3.1+ environment, an error page does not need to be
```

Comment:
This is now ambiguous. It says up until 3.0 and then 3.1+, but about 3.0? As far as I can see that improvement does apply to Servlet 3.0. See for example https://stackoverflow.com/questions/7997286/new-servlet-3-0-global-error-page-feature-does-not-work-on-tomcat-7.